AlfiyaSiddique/TastyTrails
# Feature like dislike

## What does this PR do?

Likes, disliked recipies, shows the liked recipies in the user profile, also syncs the recipie like/unlike accross all the userts
Formatted the code to look standard.

Fixes #6 



## How should this be tested?




- Like a recipe
- check the liked recipies in user profile
- login into another account to see if the liked count has increased

## Checklist



### Required
